packages/start/src/config/index.ts

```diff
@@ -18,6 +18,7 @@ import { config } from 'vinxi/plugins/config'
 import { serverFunctions } from '@vinxi/server-functions/plugin'
 // @ts-expect-error
 import { serverTransform } from '@vinxi/server-functions/server'
+import { createNitro } from 'nitropack'
```

Comment:
In the `package.json` we should move this from `devDependencies` to `dependencies`.

Reply:
Yeah, we should